packages/core/src/Stores/client-store.js

```diff
@@ -1086,6 +1111,12 @@ export default class ClientStore extends BaseStore {
             this.has_cookie_account = false;
         }
     }
+
+    @action.bound
```

Comment:
You need to update your PR to be compatible to MobX 6.
remove this and define it in `makeObservable` section

Reply:
Will resolve this